src/components/network-cost-trend-chart.tsx

```diff
@@ -0,0 +1,134 @@
+import { Box } from "@mui/material";
+import { useMemo } from "react";
+
+import type { GenericObservation, ObservationValue } from "src/domain/data";
+
+import { AxisHeightCategories } from "./charts-generic/axis/axis-height-categories";
+import { AxisWidthLinear } from "./charts-generic/axis/axis-width-linear";
+import { ChartContainer, ChartSvg } from "./charts-generic/containers";
+import { Tooltip } from "./charts-generic/interaction/tooltip";
+import { InteractionDotted } from "./charts-generic/overlay/interaction-dotted";
+import { Dots } from "./charts-generic/scatter-plot/dots";
+import { ScatterPlot } from "./charts-generic/scatter-plot/scatter-plot-state";
+import { SectionProps } from "./detail-page/card";
+
+type NetworkCostTrendChartProps = {
+  observations: GenericObservation[];
+};
+
+type CategoryKey =
+  | "SunNetworkCostsNE5"
+  | "SunNetworkCostsNE6"
+  | "SunNetworkCostsNE7";
+
+type TransformedDatum = {
+  category: string;
+  value: number;
+  SunName: string;
+  operatorLabel: ObservationValue;
+  operator: ObservationValue;
+  uniqueId: ObservationValue;
+  municipalityLabel: ObservationValue;
+  period: ObservationValue;
+};
+
+const TARGET_OPERATORS = [
+  "Elektrizitätswerk der Stadt Zürich",
+  "BKW Energie AG",
+];
+
+const CATEGORIES: { key: CategoryKey; label: string }[] = [
+  {
+    key: "SunNetworkCostsNE5",
+    label: "Medium Voltage Distribution (Level 5) - CHF./kVA",
+  },
+];
+
+export const NetworkCostTrendChart = ({
+  observations,
+  id,
+}: NetworkCostTrendChartProps & Omit<SectionProps, "entity">) => {
+  const transformedData: TransformedDatum[] = useMemo(() => {
+    return CATEGORIES.flatMap(({ key, label }) => {
+      const entries: TransformedDatum[] = [];
+
+      const all = observations
+        .map((obs) => obs[key])
+        .filter((v): v is number => !isNaN(v as number));
+      const median =
+        all.length > 0
+          ? all.sort((a, b) => a - b)[Math.floor(all.length / 2)]
+          : undefined;
+
+      if (median !== undefined) {
+        entries.push({
+          category: label,
+          value: median,
+          SunName: label,
```

Comment:
We should use label and not SunName. The PropertiesLikeThat are the one from the data, we use "lowerCase" inside the app.